### PR TITLE
Add low tanimoto check in routes.py. Display warning for tanimoto und…

### PR DESCRIPTION
**Pull request by karmenlu**
…er .6 in cluster.html. #83

### CODE REVIEW

**Review by aislinblack — CHANGES_REQUESTED** [review]
I would prefer if it warned the user BEFORE they submit the coefficient.

File: app/routes.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from app import app
-from flask import render_template, request, session
+from flask import render_template, request, session, flash
```
Comment (aislinblack) [inline app/routes.py]: do we still need to import flask?

File: app/routes.py [hunk 2]
```diff
@@ -112,6 +112,7 @@ def verify_pains_by_error():
 def final_compounds():
   good_smiles = session.get('good_smiles')
   bad_smiles = session.get('bad_smiles')
+  low_tanimoto_warning = False
```
Comment (aislinblack) [inline app/routes.py]: this doesn't seem needed anymore

**Review by aislinblack — CHANGES_REQUESTED** [review]
Just a couple changes and then you should be good to go!

File: app/routes.py (hunk 1 above)
Comment (karmenlu) [reply app/routes.py]: I will remove this.

File: app/routes.py (hunk 2 above)
Comment (karmenlu) [reply app/routes.py]: Will remove this too.

File: app/templates/pains_verify_and_coefficient_use.html [hunk 3]
```diff
@@ -59,8 +59,18 @@ <h3>Verify Pains Filter</h3>
...
       Mpo Score 6 Color: <input type="color" name="mpo6Color"/><br>
       Tanimoto Coefficient: 
-        <input type="number" name="tanimoto" step=".01" max=100 min=0/> 
-        <input type="submit" value="Cluster Compounds" >
+
+        <input type="number" id="tanimoto" name="tanimoto" step=".01" max=100 min=0/> 
+        <script>
+          function warnLow() {
+            const tanimoto = document.getElementById("tanimoto").value;
+            console.log(tanimoto);
```
Comment (aislinblack) [inline app/templates/pains_verify_and_coefficient_use.html]: maybe get rid of the console.log

**Review by aislinblack — CHANGES_REQUESTED** [review]

**Review by aislinblack — APPROVED** [review]

**Pull request merged**